src/Http/Http.Abstractions/src/StatusCodes.cs

```diff
@@ -3,77 +3,338 @@
 
 namespace Microsoft.AspNetCore.Http
 {
-    // Status Codes listed at http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
+    /// <summary>
+    /// A collection of contants for HTTP status codes.
+    ///
+    /// Status Codes listed at http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
+    /// </summary>
     public static class StatusCodes
     {
+        /// <summary>
+        /// HTTP status code 100.
+        /// </summary>
         public const int Status100Continue = 100;
```

Comment:
Good grief. I gave up when I got to this type

Reply:
A little patience and a whole lotta <kbd>Cmd</kbd>+<kbd>V</kbd> 😝 .